packages/dcos-calico/buildinfo.json

```diff
@@ -0,0 +1,19 @@
+{
+  "sources": {
+    "calico": {
+      "kind": "url",
+      "url": "https://github.com/projectcalico/cni-plugin/releases/download/v3.8.2/calico-amd64", 
```

Comment:
We cannot control the version of these binaries with the `CALICO_NODE_VERSION` parameter.